sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Key Vault Certificates client library</Description>
     <AssemblyTitle>Microsoft Azure.Security.KeyVault.Certificates client library</AssemblyTitle>
-    <VersionPrefix>4.0.0</VersionPrefix>
+    <Version>4.0.0</Version>
```

Comment:
At this point in time given the instructions @weshaggard sent out, shouldn't this be 4.0.0-preview.6? (Our last shipped preview was preview.5.)

Reply:
Fixed for now -- but I think it'll be going to 4.0.0 very shortly.